pootle/apps/pootle_store/unit/timeline.py

```diff
@@ -257,4 +295,7 @@ def get_grouped_entries(self):
         return grouped_entries
 
     def get_entry(self, item):
-        return self.entry_class(ProxySubmission(item)).entry
+        submission = ProxySubmission(item)
```

Comment:
we build comments for suggestion that is a part of `ProxySubmission` but generally speaking I'd agree to move it away somehow.